### ISSUE TITLE
Broken link to named Tracer

### ISSUE THREAD

**Issue by CrowdHailer** [open-telemetry/opentelemetry-erlang#174]
On this section a link to `Named Tracer` is broken https://hexdocs.pm/opentelemetry_api/OpenTelemetry.html#register_application_tracer/1

I think the correct target is https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md

**Comment by chulkilee** [comment]
What about using URL with tags instead of master branch? Like https://github.com/open-telemetry/opentelemetry-specification/blob/v0.7.0/specification/trace/api.md

**Comment by tsloughter** [comment]
Good point about using tags instead of  branches in these links to the spec.
